compiler/rustc_errors/src/emitter.rs

```diff
@@ -1470,9 +1467,7 @@ impl EmitterWriter {
                     let mut to_add = FxHashMap::default();
 
                     for (depth, style) in depths {
-                        if multilines.get(&depth).is_some() {
-                            multilines.remove(&depth);
```

Comment:
I'll wait for CI to complete because I'm surprised that changing these two lines don't affect the output (I'm guessing it was dead code).

Otherwise, r=me.

Reply:
If `depth` was `Some(_)`, it would have gotten `remove`d in the then branch, but `remove` *also* returns the previous value, which should explain why there is no difference. Note that `remove` returns `None` if the value wasn't there previously.